frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -5,6 +5,7 @@ import { useBackend } from "main/utils/useBackend";
 import Accordion from 'react-bootstrap/Accordion';
 import TestJobForm from "main/components/Jobs/TestJobForm";
 import JobComingSoon from "main/components/Jobs/JobComingSoon";
+import JobMilktheCows from "main/components/Jobs/JobMilktheCows";
 
 import { useBackendMutation } from "main/utils/useBackend";
 
```

Comment:
Let's also add this---it's a copy/paste/edit of the code for the test job:

```jsx
 // test job 

    const objectToAxiosParamsMilkCowsJob = (data) => ({
        url: `/api/jobs/launch/milkcows`,
        method: "POST"
    });

    // Stryker disable all
    const milkCowsJobMutation = useBackendMutation(
        objectToAxiosParamsMilkCowsJob,
        {  },
        ["/api/jobs/all"]
    );
    // Stryker enable all

    const submitMilkCowsJob = async (data) => {
        console.log("submitMilkCowsJob, data=", data);
        milkCowsJobMutation.mutate(data);
    }

```

Then, change this line: 

```jsx
form: <JobMilktheCows />
```

To this one: 

```jsx
form: <JobMilktheCows submitAction={submitMilkCowsJob}
```

That way, the entire frontend is ready to go... as soon as the backend is complete, the job will work.